usaspending_api/etl/management/commands/load_transactions_in_delta.py

```diff
@@ -57,6 +58,19 @@ class Command(BaseCommand):
     no_initial_copy: bool
     logger: logging.Logger
     spark: SparkSession
+    award_id_lookup_delete_subquery: str = """
+        SELECT aidlu.transaction_unique_id AS id_to_remove
+        FROM int.award_id_lookup AS aidlu LEFT JOIN raw.detached_award_procurement AS dap ON (
+            aidlu.detached_award_procurement_id = dap.detached_award_procurement_id
+        )
+        WHERE aidlu.detached_award_procurement_id IS NOT NULL AND dap.detached_award_procurement_id IS NULL
+        UNION ALL
+        SELECT aidlu.transaction_unique_id AS id_to_remove
+        FROM int.award_id_lookup AS aidlu LEFT JOIN raw.published_fabs AS pfabs ON (
+            aidlu.published_fabs_id = pfabs.published_fabs_id
+        )
+        WHERE aidlu.published_fabs_id IS NOT NULL AND pfabs.published_fabs_id IS NULL
+    """
```

Comment:
Any reason not to do a single join to `transaction_normalized` instead of a `UNION` of two queries to `transaction_fabs` and `transaction_fpds`?

Reply:
Nevermind, I see that his is querying the `raw` tables not the `int` tables